Cartfile

```diff
@@ -2,7 +2,7 @@ binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.6
 github "mapbox/MapboxDirections.swift" ~> 0.10
 github "52inc/Pulley" == 1.4
 github "rs/SDWebImage" ~> 4.1
-github "Project-OSRM/osrm-text-instructions.swift" ~> 0.3.2
+github "Project-OSRM/osrm-text-instructions.swift" ~> 0.3.3
```

Comment:
Can you update the podspec as well?

Reply:
The podspec specifies `~> v0.3`, so running `pod install` should fetch v0.3.3 automatically. Are you seeing otherwise?

Reply:
Ah right, good call.